cypress/e2e/settings/users/patron-block/charged.out.removed.cy.js

```diff
@@ -0,0 +1,200 @@
+import uuid from 'uuid';
```

Comment:
let's remove file name to more understandable
patron-block-maximum-items-charged-out-removed.cy.js
thx